### PR TITLE
selector headers

### PR DESCRIPTION
**Pull request by shadrackkibet**
@dannyparsons i have added selector headers for this two dialogs. This is ready for you.

### CODE REVIEW

File: instat/dlgSpells.vb [hunk 1]
```diff
@@ -54,29 +54,34 @@ Public Class dlgSpells
         ucrReceiverRainfall.Selector = ucrSelectorForSpells
         ucrReceiverRainfall.AddIncludedMetadataProperty("Climatic_Type", {Chr(34) & "rain" & Chr(34)})
         ucrReceiverRainfall.bAutoFill = True
+        ucrReceiverRainfall.strSelectorHeading = "Numerics"
```
Comment (dannyparsons) [inline instat/dlgSpells.vb]: These are actually only rainfall columns not all numerics. So I suggest "Rain Variables" here. And similar for any others with a fixed climatic type

Comment (shadrackkibet) [inline instat/dlgSpells.vb]: @dannyparsons this is now sorted.

File: instat/dlgExtremesClimatic.vb [hunk 2]
```diff
@@ -43,24 +43,29 @@ Public Class dlgExtremesClimatic
...
         ucrReceiverDOY.SetClimaticType("doy")
         ucrReceiverDOY.Selector = ucrSelectorClimaticExtremes
         ucrReceiverDOY.bAutoFill = True
+        ucrReceiverDOY.strSelectorHeading = "Numerics"
 
         ucrReceiverElement.SetParameter(New RParameter("x", 0))
         ucrReceiverElement.Selector = ucrSelectorClimaticExtremes
         ucrReceiverElement.SetParameterIsString()
         ucrReceiverElement.bWithQuotes = False
+        ucrReceiverElement.strSelectorHeading = "Numerics"
```
Comment (dannyparsons) [inline instat/dlgExtremesClimatic.vb]: Can you do the same for this dialog?

**Comment by maxwellfundi** [comment]
@shadrackkibet  is everything done here so that this can be merged by @dannyparsons ?

**Comment by shadrackkibet** [comment]
yes, this is done.

**Comment by maxwellfundi** [comment]
@shadrackkibet  great. This can now be merged @dannyparsons .

**Pull request merged**